worker/inference-mock.ts

```
@@ -0,0 +1,327 @@
+            lowShelfDb: -1.5 + Math.random() * 3,
+            highShelfDb: -0.5 + Math.random() * 2,
+            compDb: 2 + Math.random() * 4,
+            targetLufs: -14 + Math.random() * 2
+          }
+          break
+        case 'preset_cls':
+          const presets = ['default', 'vocal_focus', 'inst_focus', 'balanced', 'bright', 'warm']
+          result.presetId = presets[Math.floor(Math.random() * presets.length)]
+          break
+        case 'align_conf':
+          result.alignConfidence = 0.7 + Math.random() * 0.25
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)